package.json

```diff
@@ -24,6 +25,7 @@
     "xtend": "^4.0.0"
   },
   "devDependencies": {
-    "from": "^0.1.3"
+    "from": "^0.1.3",
+    "q": "^1.4.1"
```

Comment:
Question here, we already using `bluebird`, any reason why we need to add the `q` to devDependencies?